packages/terra-worklist-data-grid/translations/en-US.json

```diff
@@ -0,0 +1,3 @@
+{
+  "Terra.worklistDataGrid.maskedCell": "Masked data"
```

Comment:
Is there a Jira logged to add translations for the other locales?

Reply:
I need to create this JIRA for the translations.  Do you have an example JIRA that I could follow as a template?

Reply:
@cm9361  I will log the Jira and post the id once logged.

Reply:
@cm9361  @sycombs  https://jira2.cerner.com/browse/UXPLATFORM-9183